Paper: Short `wailord` poster accompaniment

HaoZeke opened the pull request:
As listed in the title. Meant to accompany a poster presentation on submission 68 titled "Wailord: Parsers and Reproducibility for Quantum Chemistry".

*Editor*: <!--editor-->@deniederhut<!--end-editor-->

*Reviewers*: <!--reviewers-list-->@vsinha027<!--end-reviewers-list-->

HaoZeke committed:
ENH: Short `wailord` poster accompaniment

scoobies commented:
Hi!, I'm @scoobies, a friendly bot.

Type ```@scoobies help``` to discover how I can help you.

HaoZeke commented:
@scoobies help

scoobies commented:
Hello @HaoZeke, here are the things you can ask me to do:

```

# List all available commands
@scoobies help

# Show our community Code of Conduct and Guidelines
@scoobies code of conduct

# Add to this issue's reviewers list
@scoobies add @username to reviewers

# Remove from this issue's reviewers list
@scoobies remove @username from reviewers

# Assign a user as the editor of this submission
@scoobies assign @username as editor

# Remove the editor assigned to this submission
@scoobies remove editor

# Add a user to this issue's assignees list
@scoobies add assignee: @username

# Remove a user from this issue's assignees list
@scoobies remove assignee: @username

# Builds paper
@scoobies build paper

# Checks build status
@scoobies build status

# Check the references of the paper for missing DOIs
# Optionally, it can be run on a non-default branch 
@scoobies check references
@scoobies check references from custom-branch-name

# Label issue with: paper
@scoobies mark paper

# Label issue with: needs-review. Remove labels: unready, needs-more-review, pending-comment, ready
@scoobies mark needs review

# Label issue with: needs-more-review. Remove labels: unready, needs-review, pending-comment, ready
@scoobies mark needs more review

# Label issue with: pending-comment. Remove labels: unready, needs-review, needs-more-review, ready
@scoobies mark pending comment

# Label issue with: ready. Remove labels: unready, needs-review, needs-more-review, pending-comment
@scoobies mark ready

# Label issue with: unready. Remove labels: needs-review, needs-more-review, pending-comment, ready
@scoobies mark not ready

# Label issue with: does_not_build:server
@scoobies mark server fail

# Remove labels: does_not_build:server
@scoobies mark server success

```

HaoZeke commented:
@scoobies build paper

scoobies commented:
building paper...

scoobies commented:
{"message"=>"Invalid paper specified", "status"=>"fail"}

HaoZeke commented:
@scoobies check references

deniederhut commented:
@scoobies mark paper

deniederhut commented:
@scoobies build status

scoobies commented:
{"HaoZeke-rgWailord"=>{"data"=>{"build_output"=>"[*] Updating proceedings build tools in {self.master_repo_path}...\nAlready up to date.\n[*] Check out paper repository...\nCloning into '/tmp/tmplg39cysp'...\nMoving proceedings build tools to temp directory...\n[*] Build the paper...\nBuilding: rohit_goswami_wailord\n*** Warning: /tmp/tmplg39cysp/publisher/../output/rohit_goswami_wailord/paper_stats.json does not exist.\n", "build_pdf_path"=>"/procbuild/procbuild/../cache/HaoZeke-rgWailord.pdf", "build_status"=>"success", "build_timestamp"=>"28/05 05:03"}, "status"=>"success"}}

deniederhut commented:
@scoobies mark needs review

deniederhut commented:
@scoobies assign @deniederhut as editor

scoobies commented:
Assigned! @deniederhut is now the editor

HaoZeke committed:
MAINT: Add more labels

HaoZeke committed:
DOC: Add funding

HaoZeke committed:
DOC: Cleanup spelling errors

HaoZeke committed:
DOC: Clarify discussions

deniederhut commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+Structure and Implementation
+----------------------------
+
+Python has grown to become the lingua-franca for much of the scientific community :cite:`oliphantPythonScientificComputing2007a,millmanPythonScientistsEngineers2011`, in no small part because of its interactive nature. In particular, the REPL (read-evaluate-print-loop) structure which has been prioritized (from IPython to Jupyter) is one of the prime motivations for the use of Python as an explor ...
```
Is there a typo in `oliphantPythonScientificComputing2007a`? I don't see it rendering correctly in the pdf

HaoZeke replied:
Yup, sorry, fixed :)

deniederhut commented:
@scoobies add @vsinha027 to reviewers

scoobies commented:
@vsinha027 added to the reviewers list!

HaoZeke committed:
MAINT: Fix broken citation

deniederhut commented:
Hi! We've moved the deadline for first reviews back one week, to June 16th, to give us a little more time to find reviewers for all the papers this year. You can see the updated full timeline at https://github.com/scipy-conference/scipy_proceedings#timeline-for-2022.

deniederhut commented:
Hi @vsinha027! How is the review coming along?

vsinha027 commented:
Hi, Working on it. Should be able to make some update tonight.

On Tue, Jun 14, 2022 at 2:41 AM Dillon Niederhut ***@***.***>
wrote:

> Hi @vsinha027 <https://github.com/vsinha027>! How is the review coming
> along?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/pull/715#issuecomment-1154588178>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACD4RUSWMSVCPPWS77L22D3VO7ILXANCNFSM5XF5EWLA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Best regards
~Vivek
-- 
Vivek Sinha, PhD
Computational Chemist and Material Scientist
C2CAT BV, Amsterdam
***@***.***
---
Guest researcher
Inorganic Systems Engg. group
Delft University of Technology, Delft
***@***.***
The Netherlands

deniederhut commented:
Hey @vsinha027 ! Friendly reminder that reviews are due tomorrow

vsinha027 commented:
@scoobies build paper

scoobies commented:
building paper...

scoobies commented:
{"data"=>{"info"=>"Build for paper HaoZeke-rgWailord scheduled.  Note that builds are only executed if the current build attempt is more than 5 minutes old."}, "status"=>"success"}

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+
+Downstream tasks for simulations of chemical systems involve questions phrased as queries or comparative measures. With that in mind, ``wailord`` generates ``pandas`` dataframes which are indistinguishable from standard machine learning information sources, to trivialize the data-munging and preparation process. The outputs of ``wailord`` represent concrete *information* and it is not meant to st ...
+
+By construction, it differs also from existing "interchange" formats as those favored by the materials data repositories like the QCArchive project and is partially close in spirit to the ``cclib`` endeavor.
```
Please add references for QCArchive and cclib

vsinha027 replied:
Please add references for QCArchive and cclib

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+
+The core user interface is depicted in Fig. [[fig:uiwail]]. The test suites cover standard usage and serve as ad-hoc tutorials. Additionally, ``jupyter`` notebooks are also able to effectively run ``wailord`` which facilitates its use over SSH connections to high-performance-computing (HPC) clusters. The user is able to describe the nature of calculations required in a simple YAML file format. A  ...
+
+.. figure:: overviewWailord.jpg
```
Figure 1 is difficult for me to comprehend honestly. Perhaps better to replace with a workflow or flowchart for wailord?

vsinha027 replied:
To the non-expert user terms like VPT2 or Orca_vis objects are difficult to understand.

vsinha027 replied:
perhaps also reconsider the use of screenshots of your command line. I would suggest to rework the Figure to make sure that all the details are clearly visible.

HaoZeke replied:
I have attempted to clarify in text some of these concerns.

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+
+Two classes form the backbone of the data-harvesting process. The intended point of interface with a user is the ``orcaExp`` class which collects information from multiple ORCA outputs and produces dataframes which include relevant metadata (theory, basis, system, etc.) along with the requested results (energy surfaces, energies, angles, geometries, frequencies, etc.). A lower level "orca visitor ...
+
+Python has a rich set of structures implemented in the standard library, which have been liberally used for structuring outputs. Furthermore, there have been efforts to convert the grammar of graphics :cite:`wilkinsonGrammarGraphics2005` and tidy-data :cite:`wickhamWelcomeTidyverse2019` approaches to the ``pandas`` package which have also been adapted internally, including strict unit adherence u ...
```
I understand that you are trying to bring out the pros of using python and pandas, but in terms of readability, this paragraph breaks the flow of the reading. The reader is transitioned from information about ORCA suddenly to Python and pandas. I would suggest making a better connection here.

HaoZeke replied:
I have added more joining information to bridge the paragraphs.

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+User Interface
+++++++++++++++
+
+The core user interface is depicted in Fig. [[fig:uiwail]]. The test suites cover standard usage and serve as ad-hoc tutorials. Additionally, ``jupyter`` notebooks are also able to effectively run ``wailord`` which facilitates its use over SSH connections to high-performance-computing (HPC) clusters. The user is able to describe the nature of calculations required in a simple YAML file format. A  ...
```
missing link to the Figure? In the pdf is see "depicted in Fig. [[fig:uiwail]]" literally printed instead of "Figure 1".

vsinha027 replied:
Possible typo: A command-line interface "can" then .. instead of "and".

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+
+The use of computational methods for chemistry is ubiquitous and few modern chemists retain the initial skepticism of the field :cite:`kohnNobelLectureElectronic1999,schaeferMethyleneParadigmComputational1986`. Machine learning has been further earmarked :cite:`meyerMachineLearningComputational2019,dralQuantumChemistryAge2020,schuttUnifyingMachineLearning2019` as an effective accelerator for comp ...
+
+Dataset bias :cite:`engstromIdentifyingStatisticalBias2020,blumRecoveringBiasedData2019,rahamanSpectralBiasNeural2019` has gained prominence in the machine learning literature, but has not yet percolated through to the chemical sciences community. At its core, the argument for dataset biases in generic machine learning problems of image and text classification, can be linked to the difficulty in  ...
```
Can you support your statement about dataset bias not percolating through chemical sciences community with some references?

vsinha027 replied:
"This is not an issue in the physical sciences at all, as the training data is always labeled without human intervention." This statement is very general and not quite true. It depends on the task at hand. Consider for example the task of identifying active drug molecules for a particular application in a dataset of organic molecules. A machine learning-based approach would need labels on the training set as "active" and "inactive" for the given application. Such data needs curation is not automatically available. Another example is spectroscopy. For example for NMR or IR spectra of complex molecules, you need an expert to assign the peaks and identify the molecule. So the data coming from the spectrometer is not labeled.

vsinha027 replied:
"There is still a strong tendency to focus on "benchmark" datasets and results" The way this sentence reads is that tendency to use benchmarks is obsolete, is that what is meant here? Perhaps consider rephrasing to connect with the next sentence. in my understanding, you want to highlight why reproducibility is important, right?

vsinha027 replied:
Actually, I do not see a clear connection between the sentences you have used. I would strongly suggest to focus on the readability of the paper. The message is mixed and confusing to me.
I would suggest first describing what kind of solutions to needed to address the issues of reproducibility and bias in chemical sciences that you have pointed to. Then describe what ORCA is and why ORCA, and what wailord does perhaps as a new paragraph for the reader.

vsinha027 replied:
Please add "Neese, F.“The ORCA program system” Wiley interdisciplinary Reviews - Computational Molecular Science, 2012,
Vol 2., Issue 1, Pages 73–78" to the citation for ORCA, since this is the paper recommended by Prof. Frank Neese for citing ORCA.

HaoZeke replied:
> "This is not an issue in the physical sciences at all, as the training data is always labeled without human intervention." This statement is very general and not quite true. It depends on the task at hand. Consider for example the task of identifying active drug molecules for a particular application in a dataset of organic molecules. A machine learning-based approach would need labels on the training set as "active" and "inactive" for the given application. Such data needs curation is not automatically available. Another example is spectroscopy. For example for NMR or IR spectra of complex molecules, you need an expert to assign the peaks and identify the molecule. So the data coming from the spectrometer is not labeled.

Thank you, this is indeed correct. I had in mind a very particular use case where progressively more accurate (and expensive) methods are used to generate molecular data (e.g. potential energy surface scans with HF, CISD, CISD(T) etc.). I have attempted to clarify this in the text.

HaoZeke replied:
> "There is still a strong tendency to focus on "benchmark" datasets and results" The way this sentence reads is that tendency to use benchmarks is obsolete, is that what is meant here? Perhaps consider rephrasing to connect with the next sentence. in my understanding, you want to highlight why reproducibility is important, right?

In this particular instance, the reproducibility is computationally infeasible, and the methods are often not clearly published so a faulty workflow might generate wrong results which remain in the literature for a long time.

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+Conclusions
+-----------
+
+We have discussed ``wailord`` in the context of generating, in a reproducible
```
please comment on what chemical insight was obtained here?

HaoZeke replied:
I have added a short description, but I am not sure discussing the details of basis sets and quantum chemistry variational methods would benefit the reader beyond this.

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+manner the structured inputs and output datasets which facilitate chemical
+insight. The formulation of bespoke datasets tailored to the study of specific
+properties across a wide range of materials at varying levels of theory has been
+shown. The test-driven-development approach is a robust methodology for
```
Please explain better what is meant by the test-driven-deveolopment approach here for the common reader who is more interested in chemical insight and has less knowledge of the software industry.

HaoZeke replied:
Adding to the implementation details so this will come as less of a surprise to the reader.

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+shown. The test-driven-development approach is a robust methodology for
+interacting with  closed source software. It is expected that the package shall
+be augmented with more workflows, in particular, with a focus on nudged elastic
+band. The methodology here has been applied to ORCA, however, the two level
```
This is a bit unclear to me. Whom is the "expectation" from? ORCA group?

HaoZeke replied:
`wailord`, will update.

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+interacting with  closed source software. It is expected that the package shall
+be augmented with more workflows, in particular, with a focus on nudged elastic
+band. The methodology here has been applied to ORCA, however, the two level
+structure is generalization to most quantum chemistry codes as well.
```
is "generalizable"

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+band. The methodology here has been applied to ORCA, however, the two level
+structure is generalization to most quantum chemistry codes as well.
+
+Importantly, we note that the ideas expressed form a design pattern for
```
It would add value to the paper to motivate wailord as a "design pattern" in the introductory section for the readers.

HaoZeke replied:
Excellent, adding a section.

vsinha027 commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,303 @@
+Importantly, we note that the ideas expressed form a design pattern for
+interacting with a plethora of computational tools in a reproducible manner. By
+defining appropriate scopes for our structured parsers, generating deterministic
+directory trees, along with a judicious use of regular expressions for output data
```
This sentence is full of jargon considering a chemistry-oriented readership. Please simplify.

HaoZeke replied:
I'm not sure how this can be simplified for the intended audience. It is true that more chemistry oriented readers might be confused but at the same time I feel like this is meant to provide a concise summary of the points discussed and cannot be compacted without needless verbiage.

vsinha027 reviewed (changes requested):
Dear Author, I read your paper with great interest. I think the issues raised by you regarding the reproducibility of data are relevant. However, I have some points of concern before I can accept the paper:
1. The issues around reproducibility need to be clearly outlined and a concise description of the solution should be provided to the reader. At the moment, I as a reader fail to comprehend the construct of the solution.
2. I do not see many novel aspects in the proposed solution. 
a. How does wailord helo derive new insights? 
b. How is it different from using python based scripts with ASE or cclib?
c. How does using wailord make data more reproducible? I could very well supply my template input files, and use an automated set of rules to validate the parameters used in the calculation. 
3. The chemical example of H2 dissociation curve, to me, seems like a fancy way of automating the generation of plots from generated data. I believe that you might have more to add to it. I would recommend adding more examples, demonstrating the utility of wailord perhaps towards enabling the generation of training data with features extracted from the output file. This would, IMHO, be more in line with the initial motivation of machine learning in chemistry that you presented in your introduction.
4. How does wailord compare with NOMAD database where I can simply upload the output files of my calculations which will then be automatically parsed with a clear route to reproducibility? https://nomad-lab.eu/services/repo-arch

In addition to the above comments, please see more specific comments added in different parts of the text.
I would recommend this paper for publication at the conference only after the suggested changes are addressed clearly. I appreciate the efforts by the author and the editor.

vsinha027 commented:
> Hey @vsinha027 ! Friendly reminder that reviews are due tomorrow

Hi, thanks for the reminder. I have added my review now. I hope it helps make the paper better and adds value for the readers.

deniederhut commented:
@scoobies mark pending comment

deniederhut commented:
Hi @HaoZeke ! Have you had a chance to read through the review comments?

HaoZeke commented:
> Hi @HaoZeke ! Have you had a chance to read through the review comments?

Yup, it's great feedback, I'll have a proper response and updates later today, I'm traveling unfortunately.

deniederhut commented:
> I'm traveling unfortunately

Thanks for letting me know -- I hope your future travels are fortunate ones!

deniederhut commented:
Hey @HaoZeke ! How are the revisions coming along?

HaoZeke committed:
TMP: Update partially from comments

HaoZeke commented:
~**EDIT**: Still making changes to the text.~

Hi @vsinha027, thank you for your many pertinent points. I will attempt to address them in order, along with the changes made to the text itself. 
> 1. The issues around reproducibility need to be clearly outlined and a concise description of the solution should be provided to the reader. At the moment, I as a reader fail to comprehend the construct of the solution.

The construct of the solution is that design practices from software engineering patterns are a valid method to enhance and mitigate many issues prevalent in the sciences. As per the suggestions in-text, I have elaborated on this in the introduction and attempted to strengthen the conclusions.

> 2. I do not see many novel aspects in the proposed solution.
>    a. How does wailord help derive new insights?

By providing the users with a robust framework to work incrementally towards a formal language for computational chemistry; i.e. the user has a framework wherein they essentially generate the skeleton of a compiler for a given computational experiment. The parser for the data, the templates (validated later) for the input, and the extraction of information from the outputs.

>    b. How is it different from using python based scripts with ASE or cclib?

The design of scripting methods is the key aspect of `wailord`. The grammar based approach towards parser generation is more robust and easier to reason about (i.e. it is the right level of abstraction) for thinking about data-formats, instead of say, the `cclib` and ASE approach of line-by-line readers.

>    c. How does using wailord make data more reproducible? I could very well supply my template input files, and use an automated set of rules to validate the parameters used in the calculation.

This is actually very much how `wailord` works to begin with. It is a framework wherein it is possible to template the input files and validate the "experiments" at runtime and ensure that the outputs are generated in a reproducible manner. There is therefore some commonality with the ASE recipes, however only at a 50-feet distance. The internal designs are vastly different. 


> 3. The chemical example of H2 dissociation curve, to me, seems like a fancy way of automating the generation of plots from generated data. I believe that you might have more to add to it. I would recommend adding more examples, demonstrating the utility of wailord perhaps towards enabling the generation of training data with features extracted from the output file. This would, IMHO, be more in line with the initial motivation of machine learning in chemistry that you presented in your introduction.

I am afraid the introduction was perhaps not as clear as I meant it to be. The machine learning aspects of chemical sciences have little to nothing to do with the practice of software engineering and solid Pythonic code design. That the user can use the parser-based approaches to feed into say, `scikit-learn` is more a matter of library/API usage than a skill to be developed or deepened by yet another layer of syntactic sugar. The utility of `wailord`, like all design patterns lies in the ability to rapidly prototype and iterate on common pain-points in computational chemistry. 

> 4. How does wailord compare with NOMAD database where I can simply upload the output files of my calculations which will then be automatically parsed with a clear route to reproducibility? https://nomad-lab.eu/services/repo-arch

The NOMAD database is a fantastic tool, and it is rather nice to have JSON based data-workflows (especially for on-line server based calculations and pipelines). Nevertheless the idea here is not to describe tools which are imperative.


I should hasten to add that the goal of presenting `wailord` is not to demonstrate new chemical results, but to demonstrate the incorporation of traditionally unused computational workflows into the study of computational chemistry.

HaoZeke commented:
@scoobies build paper

HaoZeke committed:
TMP: More partial changes

HaoZeke committed:
MAINT: Update formatting and add more flavor text

HaoZeke commented:
> Hey @HaoZeke ! How are the revisions coming along?

I have completed my first round of revisions now.

deniederhut commented:
Thanks Rohit! @vsinha027 , do you feel this paper is now ready for inclusion in the proceedings?

vsinha027 commented:
> Thanks Rohit! @vsinha027 , do you feel this paper is now ready for inclusion in the proceedings?

almost.
@HaoZeke : can you please add some justification/description of what and why ORACA (popular/free for academic use/the ubiquity of underlying techniques)

Can you double-check if the reference for ORCA "ORCA :cite:[``](https://github.com/HaoZeke/scipy_proceedings/blob/rgWailord/papers/rohit_goswami_wailord/rohit_goswami_wailord.rst#id21)neeseORCAProgramSystem2012`" is actively added? I do not see it highlighted as a link as it is for other references.

Please add a little bit of explanation and some related links to what "Grammar" is in the context of Python. I think the chemistry community would find it very useful. 

Please add some description and more importantly reference on what a "test-driven-development" approach is. I know it is common in the software community but the readership is also from the chemistry and materials community.

Once these cosmetic comments are addressed, good to go.

HaoZeke committed:
DOC: Address reviewer responses and expand grammar

HaoZeke commented:
> @HaoZeke : can you please add some justification/description of what and why ORACA (popular/free for academic use/the ubiquity of underlying techniques)

Yup, added more context.

> Can you double-check if the reference for ORCA "ORCA :cite:[``](https://github.com/HaoZeke/scipy_proceedings/blob/rgWailord/papers/rohit_goswami_wailord/rohit_goswami_wailord.rst#id21)neeseORCAProgramSystem2012`" is actively added? I do not see it highlighted as a link as it is for other references.

Sorry, this should be fixed now.
 
> Please add a little bit of explanation and some related links to what "Grammar" is in the context of Python. I think the chemistry community would find it very useful.

I have expanded the section on the grammar, with references and the methodology by which production rules are applied.
 
> Please add some description and more importantly reference on what a "test-driven-development" approach is. I know it is common in the software community but the readership is also from the chemistry and materials community.

I have added some references, however, I opted to not cite books on software management mostly because a lot of them (e.g. Agile practices) are not a good idea for most academic projects.

> Once these cosmetic comments are addressed, good to go.

I hope this is satisfactory now (and am happy to make more changes). This is much improved with your inputs.

HaoZeke commented:
@scoobies build paper

scoobies commented:
building paper...

scoobies commented:
{"data"=>{"info"=>"Build for paper HaoZeke-rgWailord scheduled.  Note that builds are only executed if the current build attempt is more than 5 minutes old."}, "status"=>"success"}

HaoZeke committed:
MAINT: Fix formatting

deniederhut commented:
Thanks for the fast turnaround @HaoZeke ! @vsinha027 , are happy with the recent revisions?

deniederhut commented:
@scoobies build status

scoobies commented:
{"HaoZeke-rgWailord"=>{"data"=>{"build_output"=>"[*] Updating proceedings build tools in {self.master_repo_path}...\nAlready up to date.\n[*] Check out paper repository...\nCloning into '/tmp/tmpivrtn02u'...\nMoving proceedings build tools to temp directory...\n[*] Build the paper...\nBuilding: rohit_goswami_wailord\n*** Warning: /tmp/tmpivrtn02u/publisher/../output/rohit_goswami_wailord/paper_stats.json does not exist.\nError compiling BiBTeX\nbibtex error output:\n================================================================================\nb'This is BibTeX, Version 0.99d (TeX Live 2017/Debian)\\nThe top-level auxiliary file: paper.aux\\nThe style file: alphaurl.bst\\nDatabase file #1: SciPyCon2022.bib\\nWarning--entry type for \"communityTuringWayHandbook2019\" isn\\'t style-file defined\\n--line 72 of file SciPyCon2022.bib\\nWarning--empty year in ahoCompilersPrinciplesTechniques2007\\nWarning--empty year in bhatEvaluatingEfficacyTestdriven2006\\nWarning--empty note in blumRecoveringBiasedData2019\\nWarning--empty journal in deComparingMoleculesSolids2016\\nWarning--empty year in deComparingMoleculesSolids2016\\nWarning--empty journal in desaiSurveyEvidenceTestdriven2008\\nWarning--empty year in desaiSurveyEvidenceTestdriven2008\\nWarning--empty journal in dralQuantumChemistryAge2020\\nWarning--empty year in dralQuantumChemistryAge2020\\nWarning--empty note in engstromIdentifyingStatisticalBias2020\\nWarning--empty journal in gaoMachineLearningCorrection2016\\nWarning--empty year in gaoMachineLearningCorrection2016\\nWarning--empty journal in hafnerAbinitioSimulationsMaterials2008\\nWarning--empty year in hafnerAbinitioSimulationsMaterials2008\\nWarning--empty note in hojaQM7XComprehensiveDataset2020\\nWarning--empty note in huberAiiDAScalableComputational2020\\nWarning--empty journal in kohnNobelLectureElectronic1999\\nWarning--empty year in kohnNobelLectureElectronic1999\\nWarning--empty journal in kolosImprovedTheoreticalGround1968\\nWarning--empty year in kolosImprovedTheoreticalGround1968\\nWarning--empty journal in larsenAtomicSimulationEnvironment2017\\nWarning--empty year in larsenAtomicSimulationEnvironment2017\\nWarning--empty journal in millmanPythonScientistsEngineers2011\\nWarning--empty year in millmanPythonScientistsEngineers2011\\nWarning--empty journal in meyerMachineLearningComputational2019\\nWarning--empty year in meyerMachineLearningComputational2019\\nWarning--empty journal in neeseORCAProgramSystem2012\\nWarning--empty year in neeseORCAProgramSystem2012\\nWarning--empty journal in oliphantPythonScientificComputing2007\\nWarning--empty year in oliphantPythonScientificComputing2007\\nWarning--empty journal in oboyleCclibLibraryPackageindependent2008a\\nWarning--empty year in oboyleCclibLibraryPackageindependent2008a\\nWarning--empty journal in pizziAiiDAAutomatedInteractive2016\\nWarning--empty year in pizziAiiDAAutomatedInteractive2016\\nWarning--empty journal in pengReproducibleResearchComputational2011\\nWarning--empty year in pengReproducibleResearchComputational2011\\nWarning--empty note in rahamanSpectralBiasNeural2019\\nWarning--empty journal in smithMolSSIQCArchiveProject2021\\nWarning--empty year in smithMolSSIQCArchiveProject2021\\nWarning--empty journal in schaeferMethyleneParadigmComputational1986\\nWarning--empty year in schaeferMethyleneParadigmComputational1986\\nWarning--empty journal in seniorProteinStructurePrediction2019\\nWarning--empty year in seniorProteinStructurePrediction2019\\nWarning--empty journal in schuttUnifyingMachineLearning2019\\nWarning--empty year in schuttUnifyingMachineLearning2019\\nWarning--empty journal in sandveTenSimpleRules2013\\nWarning--empty year in sandveTenSimpleRules2013\\nWarning--empty journal in wickhamWelcomeTidyverse2019\\nWarning--empty year in wickhamWelcomeTidyverse2019\\nWarning--empty year in wilkinsonGrammarGraphics2005\\nWarning--empty year in asgeirssonExploringPotentialEnergy2018\\n(There were 52 warnings)\\nThis is pdfTeX, Version 3.14159265-2.6-1.40.18 (TeX Live 2017/Debian) (preloaded format=pdflatex)\\n restricted \\\\write18 enabled.\\nentering extended mode\\n(./paper.tex\\nLaTeX2e <2017-04-15>\\nBabel <3.18> and hyphenation patterns for 3 language(s) loaded.\\n(./IEEEtran.cls\\nDocument Class: IEEEtran 2015/08/26 V1.8b by Michael Shell\\n-- See the \"IEEEtran_HOWTO\" manual for usage information.\\n-- http://www.michaelshell.org/tex/ieeetran/\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ppl.fd)\\n-- Using IEEE Computer Society mode.\\n-- Using 8.5in x 11in (letter) paper.\\n-- Using PDF output.\\n-- This is a 10 point document.\\n) (/usr/share/texlive/texmf-dist/tex/latex/cmap/cmap.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ifthen.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/fontenc.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/base/t1enc.def)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1ppl.fd)<<t1.cmap>>)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/inputenc.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/base/utf8.def\\n(/usr/share/texlive/texmf-dist/tex/latex/base/t1enc.dfu)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ot1enc.dfu)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/omsenc.dfu)))\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/caption.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/caption3.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/keyval.sty))\\n\\nPackage caption Warning: Unsupported document class (or package) detected,\\n(caption)                usage of the caption package is not recommended.\\nSee the caption package documentation for explanation.\\n\\n) (/usr/share/texlive/texmf-dist/tex/latex/float/float.sty) (./scipy.sty\\n(./status.sty (./draftwatermark.sty (./everypage.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphicx.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphics.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/trig.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/graphics.cfg)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-def/pdftex.def)))\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/color.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/color.cfg))))\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/mathptmx.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/helvet.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/courier.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsmath.sty\\nFor additional information on amsmath, use the `?\\' option.\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amstext.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsgen.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsbsy.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsopn.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amsfonts.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amssymb.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/bm.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hyperref.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/hobsub-hyperref.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/hobsub-generic.sty))\\n(/usr/share/texlive/texmf-dist/tex/generic/ifxetex/ifxetex.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/auxhook.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/kvoptions.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/pd1enc.def)\\n(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/hyperref.cfg)\\n(/usr/share/texlive/texmf-dist/tex/latex/url/url.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hpdftex.def\\n(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/rerunfilecheck.sty))\\n\\nPackage hyperref Warning: Option `pdftex\\' has already been used,\\n(hyperref)                setting the option has no effect on input line 43.\\n\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/longtable.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/multirow/multirow.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/booktabs/booktabs.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/textcomp.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ts1enc.def\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ts1enc.dfu)))\\n(/usr/share/texlive/texmf-dist/tex/latex/upquote/upquote.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/fancyvrb/fancyvrb.sty\\nStyle option: `fancyvrb\\' v2.7a, with DG/SPQR fixes, and firstline=lastline fix \\n<2008/02/07> (tvz))) (./paper.aux)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ts1cmr.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1ptm.fd)\\n-- Lines per column: 61 (exact).\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/ltcaption.sty)\\n(/usr/share/texlive/texmf-dist/tex/context/base/mkii/supp-pdf.mkii\\n[Loading MPS to PDF converter (version 2006.09.02).]\\n) (/usr/share/texlive/texmf-dist/tex/latex/oberdiek/epstopdf-base.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/grfext.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg))\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/nameref.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/gettitlestring.sty))\\n(./paper.out) (./paper.out)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1phv.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ztmcm.fd)<<ot1.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omlztmcm.fd)<<oml.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omsztmcm.fd)<<oms.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omxztmcm.fd)<<omx.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ptm.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/upzd.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ts1ptm.fd) (./page_numbers.tex)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1pcr.fd)\\n\\nLaTeX Warning: Citation `kohnNobelLectureElectronic1999\\' on page 1 undefined on\\n input line 165.\\n\\n\\nLaTeX Warning: Citation `schaeferMethyleneParadigmComputational1986\\' on page 1 \\nundefined on input line 165.\\n\\n\\nLaTeX Warning: Citation `meyerMachineLearningComputational2019\\' on page 1 undef\\nined on input line 167.\\n\\n\\nLaTeX Warning: Citation `dralQuantumChemistryAge2020\\' on page 1 undefined on in\\nput line 167.\\n\\n\\nLaTeX Warning: Citation `schuttUnifyingMachineLearning2019\\' on page 1 undefined\\n on input line 167.\\n\\n\\nLaTeX Warning: Citation `gaoMachineLearningCorrection2016\\' on page 1 undefined \\non input line 169.\\n\\n\\nLaTeX Warning: Citation `deComparingMoleculesSolids2016\\' on page 1 undefined on\\n input line 170.\\n\\n\\nLaTeX Warning: Citation `asgeirssonExploringPotentialEnergy2018\\' on page 1 unde\\nfined on input line 171.\\n\\n\\nLaTeX Warning: Citation `pengReproducibleResearchComputational2011\\' on page 1 u\\nndefined on input line 175.\\n\\n\\nLaTeX Warning: Citation `sandveTenSimpleRules2013\\' on page 1 undefined on input\\n line 175.\\n\\n\\nLaTeX Warning: Citation `communityTuringWayHandbook2019\\' on page 1 undefined on\\n input line 177.\\n\\n\\nLaTeX Warning: Citation `pizziAiiDAAutomatedInteractive2016\\' on page 1 undefine\\nd on input line 179.\\n\\n\\nLaTeX Warning: Citation `huberAiiDAScalableComputational2020\\' on page 1 undefin\\ned on input line 179.\\n\\n\\nLaTeX Warning: Citation `engstromIdentifyingStatisticalBias2020\\' on page 1 unde\\nfined on input line 186.\\n\\n\\nLaTeX Warning: Citation `blumRecoveringBiasedData2019\\' on page 1 undefined on i\\nnput line 186.\\n\\n\\nLaTeX Warning: Citation `rahamanSpectralBiasNeural2019\\' on page 1 undefined on \\ninput line 186.\\n\\n\\nLaTeX Warning: Citation `hojaQM7XComprehensiveDataset2020\\' on page 1 undefined \\non input line 196.\\n\\n\\nLaTeX Warning: Citation `seniorProteinStructurePrediction2019\\' on page 1 undefi\\nned on input line 196.\\n\\n\\nLaTeX Warning: Citation `neeseORCAProgramSystem2012\\' on page 1 undefined on inp\\nut line 203.\\n\\n\\nLaTeX Warning: Citation `hafnerAbinitioSimulationsMaterials2008\\' on page 1 unde\\nfined on input line 209.\\n\\n\\nLaTeX Warning: Citation `oliphantPythonScientificComputing2007\\' on page 1 undef\\nined on input line 225.\\n\\n\\nLaTeX Warning: Citation `millmanPythonScientistsEngineers2011\\' on page 1 undefi\\nned on input line 225.\\n\\n[1{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}]\\n\\nLaTeX Warning: Citation `larsenAtomicSimulationEnvironment2017\\' on page 2 undef\\nined on input line 251.\\n\\n\\nLaTeX Warning: Citation `huberAiiDAScalableComputational2020\\' on page 2 undefin\\ned on input line 253.\\n\\n\\nLaTeX Warning: Citation `smithMolSSIQCArchiveProject2021\\' on page 2 undefined o\\nn input line 257.\\n\\n\\nLaTeX Warning: Citation `oboyleCclibLibraryPackageindependent2008a\\' on page 2 u\\nndefined on input line 258.\\n\\n\\nLaTeX Warning: Citation `wilkinsonGrammarGraphics2005\\' on page 2 undefined on i\\nnput line 286.\\n\\n\\nLaTeX Warning: Citation `wickhamWelcomeTidyverse2019\\' on page 2 undefined on in\\nput line 287.\\n\\n\\nLaTeX Warning: Citation `desaiSurveyEvidenceTestdriven2008\\' on page 2 undefined\\n on input line 296.\\n\\n\\nLaTeX Warning: Citation `bhatEvaluatingEfficacyTestdriven2006\\' on page 2 undefi\\nned on input line 297.\\n\\n\\nLaTeX Warning: No positions in optional float specifier.\\n               Default added (so using `tbp\\') on input line 318.\\n\\n\\nUnderfull \\\\vbox (badness 2269) has occurred while \\\\output is active\\n\\nLaTeX Warning: Citation `ahoCompilersPrinciplesTechniques2007\\' on page 2 undefi\\nned on input line 348.\\n\\n[2 <./overviewWailord.jpg>]\\n\\nLaTeX Warning: Citation `kolosImprovedTheoreticalGround1968\\' on page 3 undefine\\nd on input line 479.\\n\\n[3] (/usr/share/texlive/texmf-dist/tex/latex/psnfss/ts1pcr.fd)\\n\\nLaTeX Warning: No positions in optional float specifier.\\n               Default added (so using `tbp\\') on input line 578.\\n\\n(./paper.bbl [4 <./plotH2A.png>]\\nUnderfull \\\\hbox (badness 4013) in paragraph at lines 76--82\\n[]\\\\T1/ptm/m/n/8.03 J\\xfcrgen Hafner.  Ab-initio sim-u-la-tions of ma-te-ri-als us-\\ning\\n\\nUnderfull \\\\hbox (badness 1365) in paragraph at lines 76--82\\n\\\\T1/ptm/m/n/8.03 VASP: Density-functional the-ory and be-yond.  29(13):2044--\\n\\nUnderfull \\\\hbox (badness 4217) in paragraph at lines 154--159\\n[]\\\\T1/ptm/m/n/8.03 Frank Neese.  The ORCA pro-gram sys-tem.  2(1):73--\\n\\nUnderfull \\\\hbox (badness 10000) in paragraph at lines 175--184\\n[]\\\\T1/ptm/m/n/8.03 Giovanni Pizzi, An-drea Ce-pel-lotti, Ric-cardo Saba-\\n\\nUnderfull \\\\hbox (badness 10000) in paragraph at lines 175--184\\n\\\\T1/ptm/m/n/8.03 tini, Nicola Marzari, and Boris Kozin-sky.  Ai-iDA:\\n\\nUnderfull \\\\hbox (badness 10000) in paragraph at lines 175--184\\n\\\\T1/ptm/m/n/8.03 Au-to-mated in-ter-ac-tive in-fras-truc-ture and database for\\n\\nUnderfull \\\\hbox (badness 10000) in paragraph at lines 175--184\\n\\\\T1/ptm/m/n/8.03 com-pu-ta-tional sci-ence.  111:218--230.  URL: [][]$http :\\n\\nUnderfull \\\\hbox (badness 10000) in paragraph at lines 175--184\\n\\\\T1/ptm/m/n/8.03 / / www . sciencedirect . com / science / article / pii / S092\\n7025615005820$[][],\\n\\nUnderfull \\\\hbox (badness 10000) in paragraph at lines 210--220\\n\\\\T1/ptm/m/n/8.03 Quan-tum Chem-istry.  231(4742):1100--1107.  URL:\\n\\nUnderfull \\\\hbox (badness 10000) in paragraph at lines 210--220\\n[][]$\\\\T1/ptm/m/n/8.03 10 . 1126 / science . 231 . 4742 . 1100$[][], [][]$\\\\T1/pc\\nr/m/n/8.03 arXiv : 17818539 ; http :\\n\\nUnderfull \\\\hbox (badness 10000) in paragraph at lines 210--220\\n\\\\T1/pcr/m/n/8.03 / / web . archive . org / web / 20200929150418 / https : / /\\n\\nUnderfull \\\\hbox (badness 10000) in paragraph at lines 210--220\\n\\\\T1/pcr/m/n/8.03 science . sciencemag . org / content / 231 / 4742 / 1100$[][]\\\\\\nT1/ptm/m/n/8.03 ,\\n\\nUnderfull \\\\hbox (badness 3482) in paragraph at lines 245--251\\n\\\\T1/ptm/m/n/8.03 9(10):e1003285.  URL: [][]$https : / / journals . plos . org /\\n ploscompbiol /\\n\\n! Package inputenc Error: Unicode char \\xc3J (U+4D)\\n(inputenc)                not set up for use with LaTeX.\\n\\nSee the inputenc package documentation for explanation.\\nType  H <return>  for immediate help.\\n ...                                              \\n                                                  \\nl.269 ...]{asgeirssonExploringPotentialEnergy2018}\\n                                                  \\n!  ==> Fatal error occurred, no output PDF file produced!\\nTranscript written on paper.log.\\n'\n================================================================================\n[X] Paper build failed.\nA build error occurred during the self.retrieve_pdf step.", "build_pdf_path"=>"", "build_status"=>"Build failed, no pdf can be found", "build_timestamp"=>"30/06 00:14"}, "status"=>"fail"}}

HaoZeke committed:
MAINT: Switch to bibtex

HaoZeke commented:
@scoobies build paper


@deniederhut sorry, forgot to switch to bibtex for the export.

HaoZeke commented:
@scoobies build status

scoobies commented:
{"HaoZeke-rgWailord"=>{"data"=>{"build_output"=>"[*] Updating proceedings build tools in {self.master_repo_path}...\nAlready up to date.\n[*] Check out paper repository...\nCloning into '/tmp/tmpmgqd6g9e'...\nMoving proceedings build tools to temp directory...\n[*] Build the paper...\nBuilding: rohit_goswami_wailord\n*** Warning: /tmp/tmpmgqd6g9e/publisher/../output/rohit_goswami_wailord/paper_stats.json does not exist.\n", "build_pdf_path"=>"/procbuild/procbuild/../cache/HaoZeke-rgWailord.pdf", "build_status"=>"success", "build_timestamp"=>"30/06 00:25"}, "status"=>"success"}}

deniederhut commented on papers/rohit_goswami_wailord/SciPyCon2022.bib:
```diff
@@ -0,0 +1,563 @@
+  ids = {asgeirssonExploringPotentialEnergy2018a},
+  title = {Exploring {{Potential Energy Surfaces}} with {{Saddle Point Searches}}},
+  booktitle = {Handbook of {{Materials Modeling}}},
+  author = {{\'A}sgeirsson, Vilhj{\'a}lmur and J{\'o}nsson, Hannes},
```
😬 PDFLatex is really mad about that Á. Could you try changing the line to `author = {{\'{A}}sgeirsson, Vilhjálmur and Jónsson, Hannes},`?

deniederhut replied:
Haha you beat me to it!

deniederhut commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,479 @@
+
+This definition maps neatly into the exact specification of an ``xyz`` file:
+
+.. code-block:: tcl
```
It looks like this code block is spilling over to the next column in the rendered PDF -- maybe it would be easier to fit in as a table or figure?
<img width="455" alt="Screen Shot 2022-06-29 at 7 46 35 PM" src="https://user-images.githubusercontent.com/6868396/176569416-aef67287-9da6-4909-91cd-3631d33a736d.png">

HaoZeke replied:
Ah, I reduced the precision (severely) to fit in the column.

deniederhut commented on papers/rohit_goswami_wailord/rohit_goswami_wailord.rst:
```diff
@@ -0,0 +1,479 @@
+    !UHF 3-21G ENERGY
+
+    %paras
+    	R = 0.4, 2.0, 33 # x-axis of H1
```
It looks like we're getting some red outlines around the = and # characters -- is that intentional?
<img width="296" alt="Screen Shot 2022-06-29 at 7 46 49 PM" src="https://user-images.githubusercontent.com/6868396/176569499-ae9288ae-0e52-4c50-bc42-d67bb704762e.png">

HaoZeke replied:
That shouldn't be happening, but I'm not sure what's causing it (something within listings/pygments perhaps)

deniederhut replied:
I think pygments might not know how to add syntax highlighting for those? I'll leave it up to you to decide if you want to try to hack around it somehow, or turn off the syntax colors for that code block, or leave as-is.

HaoZeke replied:
I'm leaning towards leaving this be. As you noted, it is a syntax highlighting error (like all "input syntax" styles for in-house code there's no real support for these in `pygments`). The only other option seems to involve messing with the preamble to disable errors in syntax highlighting [for minted](https://tex.stackexchange.com/questions/14166/red-box-drawn-around-question-mark-operator-in-minted-erlang-code) (not a good idea), and the rest of the block is rather nicely highlighted so I'd rather not leave it plain.

deniederhut replied:
mkay sounds good to me

HaoZeke committed:
MAINT: Reduce precision heavily to fit in a column

deniederhut commented:
@scoobies check references

scoobies commented:
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/978-3-319-42913-7_28-1 is OK
- 10.1145/1159733.1159787 is OK
- 10.1039/C6CP00415F is OK
- 10.1145/1383602.1383644 is OK
- 10.1021/acs.jpclett.9b03664 is OK
- 10.1186/s13321-016-0133-7 is OK
- 10.1002/jcc.21057 is OK
- 10.1103/RevModPhys.71.1253 is OK
- 10.1063/1.1669836 is OK
- 10.1088/1361-648X/aa680e is OK
- 10.1021/acs.jctc.9b00708 is OK
- 10/dc343g is OK
- 10.1002/wcms.81 is OK
- 10.1063/5.0004608 is OK
- 10.1063/5.0004608 is OK
- 10.1002/jcc.20823 is OK
- 10/fjzzc8 is OK
- 10/fdv356 is OK
- 10.1016/j.commatsci.2015.09.013 is OK
- 10/pjb is OK
- 10.1126/science.231.4742.1100 is OK
- 10.1038/s41467-019-12875-2 is OK
- 10.1002/prot.25834 is OK
- 10.1002/wcms.1491 is OK
- 10.1038/s41592-019-0686-2 is OK
- 10.21105/joss.01686 is OK

MISSING DOIs

- 10.1038/s41597-021-00812-2 may be a valid DOI for title: QM7-X: A Comprehensive Dataset of Quantum-Mechanical Properties Spanning the Chemical Space of Small Organic Molecules

INVALID DOIs

- None
```

HaoZeke committed:
MAINT: Add a missing DOI

deniederhut commented on papers/rohit_goswami_wailord/SciPyCon2022.bib:
Would you mind citing the Nature article instead of the arxiv preprint here? https://www.nature.com/articles/s41597-021-00812-2

deniederhut commented on papers/rohit_goswami_wailord/SciPyCon2022.bib:
Looks like there is a Nature article for this one too https://www.nature.com/articles/s41597-020-00638-4#citeas

deniederhut commented on papers/rohit_goswami_wailord/SciPyCon2022.bib:
Looks like this ended up in ICML? https://proceedings.mlr.press/v97/rahaman19a.html

HaoZeke committed:
MAINT: Update DOIs

Co-authored-by: Dillon Niederhut <dillon.niederhut@gmail.com>

HaoZeke committed:
DOC: Add a fig marker

HaoZeke committed:
MAINT: Strip filenames from bib

deniederhut reviewed (approved):
😆 you are fixing this paper faster than I can review it! I think we're good to go here -- thanks Rohit and Vivek!

deniederhut commented:
@scoobies mark ready
